packages/mongodb-chatbot-server/src/services/ChatLlm.ts

```diff
@@ -73,15 +104,44 @@ export interface CallToolResponse {
   references?: Reference[];
 }
 
+export type ToolCallDirective = FunctionCallPreset | FunctionName;
+
+export interface LlmCallToolParams {
```

Comment:
Difference between `LlmCallToolParams` and `ToolCallParams`? These names are kind of confusing.